### PR TITLE
feat: XAI SDK notebooks

### PR DESCRIPTION
**Pull request by andrewferlitsch**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks
- [x] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/master/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.

### CODE REVIEW

File: notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb [hunk 1]
```diff
@@ -0,0 +1,1694 @@
...
+        "- virtualenv\n",
+        "- Jupyter notebook running in a virtual environment with Python 3\n",
+        "\n",
+        "The Cloud Storage guide to [Setting up a Python development environment](https://cloud.google.com/python/setup) and the [Jupyter installation guide](https://jupyter.org/install) provide detailed instructions for meeting these requirements. The following steps provide a condensed set of instructions:\n",
+        "\n",
+        "1. [Install and initialize the SDK](https://cloud.google.com/sdk/docs/).\n",
+        "\n",
+        "2. [Install Python 3](https://cloud.google.com/python/setup#installing_python).\n",
+        "\n",
+        "3. [Install virtualenv](Ihttps://cloud.google.com/python/setup#installing_and_using_virtualenv) and create a virtual environment that uses Python 3.\n",
```
Comment (shuesc1) [inline notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb:143]: [Install virtualenv](https://cloud.google.com/python/setup#installing_and_using_virtualenv)

File: notebooks/official/explainable_ai/sdk_custom_tabular_regression_online_explain.ipynb [hunk 2]
```diff
@@ -0,0 +1,1826 @@
...
+        "- virtualenv\n",
+        "- Jupyter notebook running in a virtual environment with Python 3\n",
+        "\n",
+        "The Cloud Storage guide to [Setting up a Python development environment](https://cloud.google.com/python/setup) and the [Jupyter installation guide](https://jupyter.org/install) provide detailed instructions for meeting these requirements. The following steps provide a condensed set of instructions:\n",
+        "\n",
+        "1. [Install and initialize the SDK](https://cloud.google.com/sdk/docs/).\n",
+        "\n",
+        "2. [Install Python 3](https://cloud.google.com/python/setup#installing_python).\n",
+        "\n",
+        "3. [Install virtualenv](Ihttps://cloud.google.com/python/setup#installing_and_using_virtualenv) and create a virtual environment that uses Python 3.\n",
```
Comment (shuesc1) [inline notebooks/official/explainable_ai/sdk_custom_tabular_regression_online_explain.ipynb:145]: [Install virtualenv](https://cloud.google.com/python/setup#installing_and_using_virtualenv)

File: notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb [hunk 3]
```diff
@@ -0,0 +1,1043 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "overview:automl,xai"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "\n",
+        "This tutorial demonstrates how to use the Vertex SDK to create tabular binary classification models and do batch prediction with explanation using Google Cloud's [AutoML](https://cloud.google.com/vertex-ai/docs/start/automl-users)."
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb:64]: updated

File: notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb [hunk 4]
```diff
@@ -0,0 +1,1043 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dataset:bank,lbn"
+      },
+      "source": [
+        "### Dataset\n",
+        "\n",
+        "The dataset used for this tutorial is the [Bank Marketing](gs://cloud-ml-tables-data/bank-marketing.csv). This dataset does not require any feature engineering. The version of the dataset you will use in this tutorial is stored in a public Cloud Storage bucket."
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb:75]: updated

File: notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb [hunk 5]
```diff
@@ -0,0 +1,1043 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "objective:automl,training,batch_prediction,xai"
+      },
+      "source": [
+        "### Objective\n",
+        "\n",
+        "In this tutorial, you create an AutoML tabular binary classification model from a Python script, and then do a batch prediction with explainability using the Vertex SDK. You can alternatively create and deploy models using the `gcloud` command-line tool or online using the Cloud Console.\n",
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb:86]: updated

File: notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb [hunk 6]
```diff
@@ -0,0 +1,1043 @@
...
+        "- virtualenv\n",
+        "- Jupyter notebook running in a virtual environment with Python 3\n",
+        "\n",
+        "The Cloud Storage guide to [Setting up a Python development environment](https://cloud.google.com/python/setup) and the [Jupyter installation guide](https://jupyter.org/install) provide detailed instructions for meeting these requirements. The following steps provide a condensed set of instructions:\n",
+        "\n",
+        "1. [Install and initialize the SDK](https://cloud.google.com/sdk/docs/).\n",
+        "\n",
+        "2. [Install Python 3](https://cloud.google.com/python/setup#installing_python).\n",
+        "\n",
+        "3. [Install virtualenv](Ihttps://cloud.google.com/python/setup#installing_and_using_virtualenv) and create a virtual environment that uses Python 3.\n",
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb:146]: fixed

File: notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb [hunk 7]
```diff
@@ -0,0 +1,1043 @@
...
+        "\n",
+        "### Set up your Google Cloud project\n",
+        "\n",
+        "**The following steps are required, regardless of your notebook environment.**\n",
+        "\n",
+        "1. [Select or create a Google Cloud project](https://console.cloud.google.com/cloud-resource-manager). When you first create an account, you get a $300 free credit towards your compute/storage costs.\n",
+        "\n",
+        "2. [Make sure that billing is enabled for your project.](https://cloud.google.com/billing/docs/how-to/modify-project)\n",
+        "\n",
+        "3. [Enable the Vertex AI APIs, Compute Engine APIs, and Cloud Storage.](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component,storage-component.googleapis.com)\n",
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb:266]: updated

File: notebooks/official/explainable_ai/sdk_custom_image_classification_online_explain.ipynb [hunk 8]
```diff
@@ -0,0 +1,1860 @@
...
+        "- virtualenv\n",
+        "- Jupyter notebook running in a virtual environment with Python 3\n",
+        "\n",
+        "The Cloud Storage guide to [Setting up a Python development environment](https://cloud.google.com/python/setup) and the [Jupyter installation guide](https://jupyter.org/install) provide detailed instructions for meeting these requirements. The following steps provide a condensed set of instructions:\n",
+        "\n",
+        "1. [Install and initialize the SDK](https://cloud.google.com/sdk/docs/).\n",
+        "\n",
+        "2. [Install Python 3](https://cloud.google.com/python/setup#installing_python).\n",
+        "\n",
+        "3. [Install virtualenv](Ihttps://cloud.google.com/python/setup#installing_and_using_virtualenv) and create a virtual environment that uses Python 3.\n",
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_online_explain.ipynb:145]: fixed

File: notebooks/official/explainable_ai/sdk_custom_image_classification_online_explain.ipynb [hunk 9]
```diff
@@ -0,0 +1,1860 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "visualize_image_explanations"
+      },
+      "source": [
+        "### Visualize the images with AI Explanations\n",
+        "\n",
+        "The images returned show the explanations for only the top class predicted by the model. This means that if one of the model's predictions is incorrect, the pixels you see highlighted are for the *incorrect class*. For example, if the model predicted \"airplane\" when it should have predicted \"cat\", you can see explanations for why the model classified this image as an airplane.\n",
```
[the remark this answers is not in the document]
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_custom_image_classification_online_explain.ipynb:1709]: thx

File: notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb (hunk 1 above)
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_custom_tabular_regression_batch_explain.ipynb:143]: fixed

File: notebooks/official/explainable_ai/sdk_custom_tabular_regression_online_explain.ipynb (hunk 2 above)
Comment (andrewferlitsch) [reply notebooks/official/explainable_ai/sdk_custom_tabular_regression_online_explain.ipynb:145]: fixed

**Review by shuesc1 — APPROVED** [review]
/lgtm

**Review by ivanmkc — APPROVED** [review]
LGTM

**Pull request merged**